misc/lldb_rb/commands/heap_page_command.py

```diff
@@ -0,0 +1,38 @@
+import lldb
+
+from lldb_rb.constants import *
+from lldb_rb.rb_base_command import RbBaseCommand
+
+class HeapPageCommand(RbBaseCommand):
+    program = "heap_page"
+    help_string = "prints out 'struct heap_page' for a VALUE pointer in the page"
+
+    def __init__(self, debugger, _internal_dict):
+        self.internal_dict = _internal_dict
+
+    def __call__(self, debugger, command, exe_ctx, result):
+        super().__call__(debugger, command, exe_ctx, result)
+        target, _process, _thread, frame = self.build_environment(debugger)
+
+        self.tHeapPageBody = target.FindFirstType("struct heap_page_body")
+        self.tHeapPagePtr = target.FindFirstType("struct heap_page").GetPointerType()
+
+        page = self._get_page(target, frame.EvaluateExpression(command))
+        page.Cast(self.tHeapPagePtr)
+
+        self._append_command_output(debugger, "p (struct heap_page *) %0#x" % page.GetValueAsUnsigned(), result)
+        self._append_command_output(debugger, "p *(struct heap_page *) %0#x" % page.GetValueAsUnsigned(), result)
+
+    def get_short_help(self):
+        return self.__class__.help_string
```

Comment:
Why is it that only `HeapPageCommand` has this method? How is the `help_string` used in `RclassExtCommand` or `TestCommand`?

Reply:
This was an optional part of the API, that I guess I didn't flesh out in a consistent way. I've since pushed both `get_short_help` and `get_long_help` up into `RbBaseCommand` and just rely on `help_string` in our command classes